sources/web/datalab/polymer/components/datalab-settings/datalab-settings.ts

```diff
@@ -90,7 +95,8 @@ class SettingsElement extends Polymer.Element {
     // TODO: Show success/error status to user
     this._busy = true;
     return SettingsManager.setUserSettingAsync('idleTimeoutInterval', this.idleTimeoutInterval)
-      .catch((e: Error) => console.log('Error updating idle timeout interval: ', e))
+      .then(() => this._idleTimeoutUpdateStatus = 'Idle timeout updated')
```

Comment:
Can we also add an error css class that makes the color dark red or brown, and gets added and cleared here? Success can still use the foreground color, but error needs to standout.